src/App.jsx

```diff
@@ -1,33 +1,65 @@
+import { useState } from 'react';
 import './App.scss';
+// import { set } from 'cypress/types/lodash';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+const TITLE = 'title';
+const DESCRIPTION = 'description';
+
+function getMoviesToRender(movies, filterValue) {
+  let preparedMovies = movies;
+
+  if (filterValue) {
+    preparedMovies = movies.filter(movie => (
+      getPreparedMivieSection(movie[TITLE], filterValue)
+      || getPreparedMivieSection(movie[DESCRIPTION], filterValue)
+    ));
+  }
+
+  return preparedMovies;
+}
+
+function getPreparedMivieSection(movieSection, filterValue) {
+  const normalizedFilterValue = filterValue.trim().toLowerCase();
+
+  return movieSection.toLowerCase().includes(normalizedFilterValue);
+}
```

Comment:
The `includes` method returns `true` or `false`, not an object, or in our case, a `section`. 
Therefore, the function name is not relevant to the result.
```suggestion
function hasQueryInContent(content, query) {
  const normalizedQuery= query.trim().toLowerCase();

  return content.toLowerCase().includes(normalizedQuery);
}
```